content/posts/2021/pangeo-goes-live/index.md

```diff
@@ -0,0 +1,48 @@
+---
+title: "Pangeo goes live on 2i2c!"
+subtitle: ""
+summary: "2i2c are pleased to announce that the first Pangeo JupyterHub is now live on 2i2c-operated infrastructure! :tada: "
+authors: ["Sarah Gibson"]
+tags: [2i2c, pangeo]
+categories: [updates]
+date: 2021-11-08
+featured: false
+draft: false
+---
+
+We have recently finished re-creating the Pangeo community JupyterHub hosted on GCP in the [2i2c-org/infrastructure](https://github.com/2i2c-org/infrastructure) repository.
+This is a huge milestone in our partnership with Pangeo to provide expertise and operations of cloud-based, vendor-agnostic Jupyter infrastructure and workflows.
+
+For users of Pangeo's infrastructure, the switch should be a smooth one!
+The new hub should behave nearly identically to the old one, and will be managed by 2i2c engineers moving forward, in partnership with the Pangeo community.
+It will be available at the same URL (https://us-central1-b.gcp.pangeo.io) and your home directories were synced to the new hub only a few days before the migration took place.
```

Comment:
I switch tenses halfway through this sentence (from future to past) 😕

Reply:
```suggestion
It will be available at the same URL (https://us-central1-b.gcp.pangeo.io) and there's no need to worry about your home directories, they were synced to the new hub only a few days before the migration took place.
```

Should resolve the tenses issue